.github/workflows/rust.yml

```diff
@@ -27,7 +27,21 @@ jobs:
         run: cargo run -- manifests --crd-dir kustomize/crd/bases
       - name: Diff
         run: test -z "$(git status --porcelain)" || (echo 'Changes detected after generating manifests'; git status; git --no-pager diff; false)
-      - name: Run tests
+      - name: Run unit tests
+        run: cargo test --lib
+
+  integration_tests:
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+      - uses: Swatinem/rust-cache@a95ba195448af2da9b00fb742d14ffaaf3c21f43 # v2
+      - name: Set up Docker Buildx
```

Comment:
why do we need this?

Reply:
Is that the "Setup Docker" part specifically?

If so it's because the `local apply --dry-run render` actually _runs_ the `docker run <kubecfg_image> show ...`, so `docker` needs to be available on the machine for that to work.

Reply:
but that action installes the buildx builder in docker.

Can't we run docker containers without installing builx?

Reply:
Ah, I see what you mean :smile: 

I confused the setup steps, it looks like the runners actually already have `docker` installed :facepalm: